# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../procps/procps-autogen-no-update-potfiles.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Do not try to update pot files When running `autogen.sh` 

./autogen.sh: line 64: po/update-potfiles: No such file or directory

--- a/autogen.sh
+++ b/autogen.sh
@@ -61,7 +61,6 @@
 rm -rf autom4te.cache
 
 set -e
-po/update-potfiles
 autopoint --force $AP_OPTS
 if ! grep -q datarootdir po/Makefile.in.in; then
 	echo autopoint does not honor dataroot variable, patching.
